specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/extensions.json

```diff
@@ -0,0 +1,346 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/extensions/clustermonitoring": {
+            "put": {
+                "tags": [
+                    "Extensions"
+                ],
+                "description": "Enables the Operations Management Suite (OMS) on the HDInsight cluster.",
+                "x-ms-examples": {
+                    "Enable cluster monitoring": {
+                        "$ref": "./examples/EnableLinuxClusterMonitoring.json"
+                    }
+                },
+                "operationId": "Extension_EnableMonitoring",

```

Comment:
>Extension [](start = 32, length = 9)

should be plural - Extensions - i.e. `"operationId": "Extensions_EnableMonitoring"`

Reply:
Done.